app/.eslintrc

```diff
@@ -1,6 +1,50 @@
 {
   "rules": {
     "no-alert": "off",
-    "no-console": "off"
+    "no-console": "off",
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          "builtin",
+          "external",
+          "internal",
+          ["parent", "sibling", "index"]
+        ],
+        "pathGroups": [
+          {
+            "pattern": "react",
+            "group": "external",
+            "position": "before"
+          },
+          {
+            "pattern": "assets/**",
+            "group": "internal"
+          },
+          {
+            "pattern": "components/**",
+            "group": "internal"
+          },
```

Comment:
this doesn't scale well - but I'll review it in https://github.com/lumada-design/hv-uikit-react/pull/3764